tests/integration_tests/performance/configs/test_vsock_throughput_config_4.14.json

```diff
@@ -0,0 +1,962 @@
+{
```

Comment:
should we have a commit removing them?

Reply:
> the 4.14 files should just be renames of the old config files right?

not quite since we also added support for guest kernel 5.10 and we also added support for running on ARM.

Reply:
makes sense, but let's remove the configs that are no longer used (the ones that don't include the host kernel version in their names)